packages/kit/src/core/adapt/utils.js

```diff
@@ -47,6 +49,26 @@ export function get_utils({ cwd, config, build_data, log }) {
 					log
 				});
 			}
+		},
+
+		/** @param {{patterns: string[], log?: boolean}} options */
+		update_ignores({ patterns, log = true }) {
+			const targets = ['.gitignore', '.prettierignore', '.eslintignore'];
```

Comment:
Rather than update all three files, can we update just `.gitignore` and use it as input to prettier and eslint in the new project template? That's what this repo does if you want an example: https://github.com/sveltejs/kit/blob/b18a45c1a71f019e3ace43035fcc3d8561cd7ac9/packages/kit/package.json#L57

Reply:
sure